pkgs/development/python-modules/ghp-import/default.nix

```diff
@@ -0,0 +1,26 @@
+{ buildPythonPackage
+, dateutil
+, fetchPypi
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "ghp-import";
+  version = "1.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "wiqc4Qw3dT4miNFk12WnANrkuNefptsKLDEyuniBiU8=";
+  };
+
+  pythonImportsCheck = [ "ghp_import" ];
+
+  propagatedBuildInputs = [ dateutil ];
+
+  meta = with lib; {
+    description = "Copy your docs directly to the gh-pages branch";
+    homepage = "https://github.com/c-w/ghp-import";
```

Comment:
Homepage changed b/c username appears to have changed. The old URL redirects here 🆗